eras/conway/test-suite/cardano-ledger-conway-test.cabal

```diff
@@ -39,7 +39,7 @@ library
         cardano-ledger-conway:{cardano-ledger-conway, testlib} >=1.9 && <1.12,
         cardano-ledger-core:{cardano-ledger-core, testlib} >=1.8.1,
         cardano-ledger-allegra ^>=1.2,
-        cardano-ledger-mary ^>=1.3,
+        cardano-ledger-mary >=1.3,
```

Comment:
Not  `>= 1.4`  ?

Reply:
You are right. Thank you for catching it, as always :wink: 
Fixed